actions/transfer-case.vbs

```diff
@@ -51,84 +51,124 @@ changelog = array()
+
+    transfer_to_worker = trim(transfer_to_worker)                'formatting the information entered in the dialog
+    transfer_to_worker = Ucase(transfer_to_worker)               'making sure we are capital in all things'
+    worker_number = trim(worker_number)
+    worker_number = Ucase(worker_number)
+
+    '----------Checks that the worker or agency is valid---------- 'must find user information before transferring to account for privileged cases.
+
+    CALL navigate_to_MAXIS_screen_review_PRIV("CASE", "CURR", is_this_priv) ' need discovery on priv cases for xfer handling'
+    IF is_this_priv = TRUE THEN script_end_procedure("This case is privileged, the script will now end.")
+
+    IF transfer_to_worker = "X127CCL" THEN script_end_procedure("This case is will be transferred via an automated script after being closed for 4 months, the script will now end.")
+
+    call navigate_to_MAXIS_screen("REPT", "USER")
+    EMWriteScreen transfer_to_worker, 21, 12
+    TRANSMIT
 
-	Do
-		Do
-			err_msg = ""
-			Dialog Dialog1
-			cancel_without_confirmation
-	      	IF IsNumeric(maxis_case_number) = false or len(maxis_case_number) > 8 THEN err_msg = err_msg & vbNewLine & "Please enter a valid case number."
-			IF servicing_worker = "" or len(servicing_worker) > 7 THEN err_msg = err_msg & vbNewLine & "Please enter a valid  worker number (EX. X120ICT)."
-			IF out_of_county_checkbox <> CHECKED and in_county_checkbox <> CHECKED THEN err_msg = err_msg & vbNewLine & "Please select the type of transfer."
-			IF transfer_reason = "" THEN err_msg = err_msg & vbNewLine & "Please enter a reason for transfer."
-			IF err_msg <> "" THEN MsgBox "*** NOTICE!***" & vbNewLine & err_msg & vbNewLine
-		Loop until err_msg = ""
-		CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to password back into MAXIS
-	LOOP UNTIL are_we_passworded_out = false					'loops until user passwords back in
-		'----------Checks that the worker or agency is valid---------- 'must find user information before transferring to account for privileged cases.
-	servicing_worker = ucase(servicing_worker)
-	call navigate_to_MAXIS_screen("REPT", "USER")
-	EMWriteScreen servicing_worker, 21, 12
-	TRANSMIT
-	EMReadScreen worker_found, 15, 24, 2
-	EMReadScreen inactive_worker, 8, 7, 38
-	IF inactive_worker = "INACTIVE" THEN MsgBox "The worker or agency selected is not active. Please try again."
-	IF worker_found = "NO WORKER FOUND" THEN MsgBox "The worker or agency selected does not exist. Please try again."
-LOOP UNTIL inactive_worker <> "INACTIVE" AND worker_found <> "NO WORKER FOUND"
+    EMReadScreen error_message, 75, 24, 2
+    error_message = trim(error_message)
+    EMReadScreen inactive_worker, 8, 7, 38
+    IF inactive_worker = "INACTIVE" THEN
+        active_worker_found = false
+        MsgBox "Please review the worker and try again. " & error_message
```

Comment:
There is no guarantee that the 'error_message' has any relevant information here. The message here should be specific to the inactive worker status.